src/pages/guidelines/carbon-for-ai/index.mdx

```diff
@@ -0,0 +1,605 @@
+IBM Design Language DNA and principles.
+
+#### New color and style tokens for creating your AI experiences
+
+To accommodate for these AI styles, we are introducing a new suite of color and
+style tokens that can be found within the main Carbon themes. Using the AI
+tokens does not require a different theme or elements package. When building
+custom AI component variants or instances in your UI use these new color tokens
+to stay consistent with the AI styling. For a full list of the new tokens, go to
+[color tokens](/elements/color/tokens).
+
+<Row>
+<Column colLg={12}>
+
+<Tabs>
+
+<Tab label="Light mode">
+
+![Light mode UI example of the AI style elements](images/ai-style-elements-light.png)
+
+</Tab>
+
+<Tab label="Dark mode">
+
+![Dark mode UI example of the AI style elements](images/ai-style-elements-dark.png)
+
+</Tab>
+
+</Tabs>
+
+</Column>
+</Row>
+
+## AI explainability
+
+Explainability is a set of processes and methods that make AI decisions more
+comprehensible to humans. _For IBMers only_: To learn more about explainability,
+see
+[Design for AI: Explainability](https://w3.ibm.com/w3publisher/design-for-ai/ai-ethics/explainability).
+
+#### Well-designed explainability:
+
+- Builds trust with users
+- Improves understanding of an AI system
+- Increases usage and interaction
+- Enables product teams to gain further insights to improve the AI system
+
+#### Use the AI label as the path to explainability
+
+The AI label is the primary indicator to communicate that AI is present, and
+it’s how users can access explainability content.
+
+#### Start with summarized explanations
+
+As a first layer of explainability, use the explainability pop-over attached to
+the AI label to provide a quick, in-context option to explain AI results for a
+particular instance.
+
+<Row>
+<Column colLg={12}>
+
+<Tabs>
+
+<Tab label="Light mode">
+
+![Light mode UI example of AI explainability shown in a popover.](images/ai-explainability-light.png)
+
+</Tab>
+
+<Tab label="Dark mode">
+
+![Dark mode UI example of AI explainability shown in a popover.](images/ai-explainability-dark.png)
+
+</Tab>
+
+</Tabs>
+
+</Column>
+</Row>
+
+### Explainability resources
+
+<Row className="resource-card-group">
+<Column colLg={4} colMd={4} noGutterSm>
+  <ResourceCard
+    subTitle="Designing for Explainable AI"
+    href="https://ibm.ent.box.com/v/designingXAI"
+    >
+
+<MdxIcon name="bee" />
+
+  </ResourceCard>
+</Column>
+<Column colLg={4} colMd={4} noGutterSm>
+  <ResourceCard
+    subTitle="AI Explainability kit (Beta)"
+    href="https://www.figma.com/design/jLtMeH3RsHnDs7xF9xUS8f/Carbon-AI---AI-Details-Beta?m=auto&t=Fz8SUpQOgZSNH9Ab-6"
+    >
+
+<MdxIcon name="figma" />
+
+  </ResourceCard>
+</Column>
+</Row>
+
+## AI label
+
+<InlineNotification>
+
+**Formerly AI slug:** The AI label was previously called "AI Slug" but has been
+updated to better reflect its usage. This name change will be reflected in code
```

Comment:
Should we also mention that this name change will be reflected in both the code and `Figma` once it moves to stable in the near future?

Reply:
This is a good point, I'll add a note about the figma component as well.